LICENCE-header.txt

```diff
@@ -0,0 +1,17 @@
+====
+    This file is part of obelix.
```

Comment:
We need to have a Copyright statement in every file, I can generate these using my scripts, as promised some time ago...